src/server/game/Spells/SpellEffects.cpp

```diff
@@ -1442,7 +1442,13 @@ void Spell::EffectCreateItem(SpellEffIndex effIndex)
     if (effectHandleMode != SPELL_EFFECT_HANDLE_HIT_TARGET)
         return;
 
-    DoCreateItem(effIndex, effectInfo->ItemType);
+    std::set<uint32> bonusListIDs = sDB2Manager.GetItemBonusTree(effectInfo->ItemType, 13); // 13 is guessed tradeskill bonus mod
+
+	std::vector<int32> bonusList;
```

Comment:
tabs

Reply:
itemBonusTreeMod must not be hardcoded there, check it more

Reply:
tabs2spaces